tidy3d/plugins/adjoint/components/simulation.py

```diff
@@ -25,6 +25,15 @@
 from .geometry import JaxPolySlab, JaxGeometryGroup
 
 
+# bandwidth of adjoint source in units of freq0 if no sources and no `fwidth_adjoint` specified
+FWIDTH_FACTOR = 1.0 / 10
+
+
+# how many processors to use for server and client side adjoint
+NUM_PROC_LOCAL = 1
+NUM_PROC_SERVER = 2
```

Comment:
`NUM_PROC_SERVER` is only used in test_adjoint so no need to define it here?

Reply:
My thought was that the server side code could use it too to set default values